client/src/Components/Results.js

```diff
@@ -288,7 +293,7 @@ export default function Results(props) {
 																		className="quiz-button"
 																		onClick={() => showDetail(student)}
 																	>
-                                    Show Ditails
+                                      Show Ditails
```

Comment:
```suggestion
                                      Show Details
```

Reply:
![image](https://user-images.githubusercontent.com/48957181/93497826-cea20280-f908-11ea-8d43-f3c10b1a46cf.png)

I changed the details table for one student to be vertically
we have the score table
the big colourful table is removed

Reply:
I can not access to the exact view of small table 
it is showing just vertically

Reply:
would you prefer to have the big colourful table ?